.gitignore

```diff
@@ -122,3 +122,5 @@ ytdlp_plugins/extractor/*
 ytdlp_plugins/postprocessor/*
 !ytdlp_plugins/postprocessor/__init__.py
 !ytdlp_plugins/postprocessor/sample.py
+*.dfxp
+*.tt
```

Comment:
Revert. We will add this separately.

@gamer191 Could you add this to the cleanup PR. Make sure to also add it to makefile